contracts/external/proxies/LiquidatorProxyV1ForSoloMargin.sol

```diff
@@ -406,8 +413,65 @@ contract LiquidatorProxyV1ForSoloMargin is
         return supplyValue >= borrowValue.add(requiredMargin);
     }
 
-    // ============ Parsing Functions ============
+    // ============ Getter Functions ============
+
+    /**
+     * Gets the current total supplyValue and borrowValue for some account. Takes into account what
+     * the current index will be once updated.
```

Comment:
Is there no function on `SoloMargin` that does this for you? That's unfortunate

Reply:
Unfortunately no, we can create one on a proxy getter contract perhaps. Could help in the lib too